features/steps/thens/attributes.py

```diff
@@ -72,4 +77,14 @@ def step_impl(context, inst, field, file_or_model, values):
     elif field == "Schema" and not context.model.schema in values:
         s = context.model.schema
         if not s.lower() in values:
-            yield StepResult(expected=values, observed=s)
\ No newline at end of file
+            yield StepResult(expected=values, observed=s)
+
+
+@validate_step('The {length_attribute} of the final segment must be 0')
```

Comment:
I'd say make the '0' a function attribute, so 
`'must be {num:d}'` . Or does it have to be a float in this case?

Reply:
Yes, it has to be a float.  There's no value in making it a function attribute because of the presence of `final segment` in the statement.  Perhaps revising to `final alignment segment` will help indicate that this implementation is specific to alignments.